test/constants.ts

```diff
@@ -0,0 +1,54 @@
+import { DEFAULT_CONFIG_STORE_VERSION } from "../src/clients";
+import { randomAddress, toWei, originChainId, destinationChainId, repaymentChainId } from "./utils";
+
+export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
```

Comment:
We export this from `src/constants.ts` - worth importing it from there?

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/408/commits/ac73ec82c232c0a564f96dc2fc90e3b173809970